governance-services/latest/install/docker.md

```diff
@@ -31,16 +31,16 @@ recommended for development and test environments only.
     ```text
     services:
         alfresco:
-            #image: alfresco/alfresco-content-repository:23.1.x
-            image: quay.io/alfresco/alfresco-governance-repository-enterprise:23.1.x
+            #image: alfresco/alfresco-content-repository:23.x.x
+            image: quay.io/alfresco/alfresco-governance-repository-enterprise:23.x.x
             mem_limit: 1700m
            
     ```
 
     ```text
         share:
-            #image: alfresco/alfresco-share:23.1.x
-            image: quay.io/alfresco/alfresco-governance-share-enterprise:23.1.x
+            #image: alfresco/alfresco-share:23.x.x
+            image: quay.io/alfresco/alfresco-governance-share-enterprise:23.x.x
             mem_limit: 1g
            
     ```
```

Comment:
```suggestion
            #image: alfresco/alfresco-share:23.x.x
            image: quay.io/alfresco/alfresco-governance-share-enterprise:23.x.x
            mem_limit: 1g
           
    ```

    Replace the ACS version (for example, ACS 23.1.x) with the compatible AGS version (for example, AGS 23.1.x).
```